src/background.ts

```diff
@@ -199,14 +199,30 @@ try {
   });
 } catch (e) {
   log.error(e);
-  shell.openPath(app.getPath("userData")).finally(() => {
-    dialog.showErrorBox(
-      "設定ファイルの読み込みに失敗しました。",
-      `${app.getPath(
-        "userData"
-      )} にある config.json の名前を変えることで解決することがあります（ただし設定がすべてリセットされます）。`
-    );
-    app.exit(1);
+  app.whenReady().then(() => {
+    dialog
+      .showMessageBox({
+        type: "error",
+        title: "設定ファイルの読み込みエラー",
+        message: `設定ファイルの読み込みに失敗しました。${app.getPath(
+          "userData"
+        )} にある config.json の名前を変えることで解決することがあります（ただし設定がすべてリセットされます）。フォルダを開きますか？`,
```

Comment:
何のフォルダが開かれるかちょっとだけ説明を追加するのどうでしょう！
```suggestion
        )} にある config.json の名前を変えることで解決することがあります（ただし設定がすべてリセットされます）。設定ファイルがあるフォルダを開きますか？`,
```

Reply:
こちら適用してマージさせていただこうと思います！